packages/@webex/plugin-meetings/src/meeting/voicea-meeting.ts

```diff
@@ -81,7 +48,11 @@ export const processNewCaptions = ({data, meeting}) => {
     const newCaption = `${transcriptsPerCsis.get(csisMember)?.text ?? ''} ${text}`.trim();
 
     // eslint-disable-next-line camelcase
```

Comment:
Why is this disable needed?

Reply:
Updated it. Not sure why it was there earlier